src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -268,6 +268,18 @@ def _rgw_rm(self, svc_id):
     def _nfs_rm(self, svc_id):
         return self._rm_stateless_svc("nfs", svc_id)
 
+    @CLIWriteCommand('orchestrator nfs update',
+                     "name=svc_id,type=CephString "
+                     "name=num,type=CephInt",
+                     'Scale an NFS service')
+    @handle_exceptions
+    def _nfs_update(self, svc_id, num):
+        spec = orchestrator.StatelessServiceSpec()
+        spec.count = num
+        completion = self.update_stateless_service("nfs", svc_id, spec)
```

Comment:
So. `StatelessServiceSpec` already contains a name, but `update_stateless_service` doesn't use it, but accepts the name as a parameter. Do you think it would make sense to just use the name attribute of the spec?

Reply:
Yes -- I meant to do that when I looked at this initially.